pkg/broker/broker.go

```diff
@@ -789,14 +769,41 @@ func (a AnsibleBroker) Unbind(
 		return nil, errors.New(errMsg)
 	}
 
+	params := make(apb.Parameters)
+	provExtCreds, err := a.dao.GetExtractedCredentials(instanceUUID.String())
+	if err != nil {
+		return nil, err
+	}
+	bindExtCreds, err := a.dao.GetExtractedCredentials(bindingUUID.String())
+	if err != nil && !client.IsKeyNotFound(err) {
+		return nil, err
+	}
+	// Add the credentials to the parameters so that an APB can choose what
+	// it would like to do.
+	params[provisionCredentialsKey] = provExtCreds.Credentials
+	if bindExtCreds != nil {
+		params[bindCredentialsKey] = bindExtCreds.Credentials
+	}
 	serviceInstance, err := a.getServiceInstance(instanceUUID)
 	if err != nil {
 		a.log.Debugf("Service instance with id %s does not exist", instanceUUID.String())
 	}
+	if serviceInstance.Parameters != nil {
+		params["provision_params"] = *serviceInstance.Parameters
+	}
+	// only launch apb if we are always launching the APB.
+	if a.brokerConfig.LaunchApbOnBind {
+		err = apb.Unbind(serviceInstance, &params, a.clusterConfig, a.log)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
I would add a warning to indicate the APB was not run in the logs, making sure we are clear that nothing was launched in case somebody expects otherwise.